cog/daily_charge.py

```
@@ -23,95 +23,257 @@
             inline=False,
         )
-        self.embed.set_footer(text=f"{interaction.user.name}充電成功！")
-        await interaction.response.send_message(embed=self.embed)
-
-    async def already_charge(self, interaction):
-        self.embed = discord.Embed(color=0xFF0000)
-        if interaction.user.avatar is not None:  # 預設頭像沒有這個
-            self.embed.set_thumbnail(url=str(interaction.user.avatar))
-        self.embed.add_field(
-            name="您夠電了，明天再來!", value="⚡⚡⚡🛐🛐🛐", inline=False
+        embed.set_footer(text=f"{user.name}充電成功！")
```

Ruff (0.8.2)

String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)